packages/react-form-state/src/components/tests/Input.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {InputHTMLAttributes} from 'react';
```

Comment:
Can you match our typical input style?

Reply:
IMO it is really messy to have what amount to fixtures directly beside the actual test files.